common/test/acceptance/tests/studio/test_studio_textbooks.py

```diff
@@ -88,3 +88,74 @@ def test_pdf_viewer_a11y(self):
+            And I name the second chapter "America"
+            And I type in "america.pdf" for the second chapter asset
+            And I save the textbook
+            Then I should see a textbook named "History" with 2 chapters
+            And I click the textbook chapters
+            Then I should see a textbook named "History" with 2 chapters
+            And the first chapter should be named "Britain"
+            And the first chapter should have an asset called "britain.pdf"
+            And the second chapter should be named "America"
+            And the second chapter should have an asset called "america.pdf"
+            And I reload the page
+            Then I should see a textbook named "History" with 2 chapters
+            And I click the textbook chapters
+            Then I should see a textbook named "History" with 2 chapters
+            And the first chapter should be named "Britain"
+            And the first chapter should have an asset called "britain.pdf"
+            And the second chapter should be named "America"
+            And the second chapter should have an asset called "america.pdf"
+        """
+        textbook_data = {
+            'textbook_name': 'History',
+            'first_chapter': 'Britain',
+            'first_asset': 'britain.pdf',
+            'second_chapter': 'America',
+            'second_asset': 'america.pdf'
+        }
+        self.textbook_upload_page.set_textbook_name(textbook_data['textbook_name'])
+        self.textbook_upload_page.fill_chapter_name('first', textbook_data['first_chapter'])
+        self.textbook_upload_page.fill_chapter_asset('first', textbook_data['first_asset'])
+        self.textbook_upload_page.submit_chapter()
+        self.textbook_upload_page.fill_chapter_name('second', textbook_data['second_chapter'])
+        self.textbook_upload_page.fill_chapter_asset('second', textbook_data['second_asset'])
+        self.textbook_upload_page.click_textbook_submit_button()
+        textbook_name = self.textbook_upload_page.textbook_name
+        self.assertEqual(textbook_data['textbook_name'], textbook_name)
+        self.textbook_upload_page.toggle_chapters()
+        number_of_chapters = self.textbook_upload_page.number_of_chapters
+        self.assertEqual(2, number_of_chapters)
+        first_chapter_name = self.textbook_upload_page.get_chapter_name('first')
+        second_chapter_name = self.textbook_upload_page.get_chapter_name('second')
+        self.assertEqual(textbook_data['first_chapter'], first_chapter_name)
+        self.assertEqual(textbook_data['second_chapter'], second_chapter_name)
+        first_asset_name = self.textbook_upload_page.get_asset_name('first')
+        second_asset_name = self.textbook_upload_page.get_asset_name('second')
+        self.assertEqual(textbook_data['first_asset'], first_asset_name)
+        self.assertEqual(textbook_data['second_asset'], second_asset_name)
+        self.textbook_upload_page.refresh_and_wait_for_load()
```

Comment:
It looks like everything after this is a repeat of the assertions above to make sure that the data doesn't change after a page reload.  Could all that be broken out into a helper method like "_assert_textbook_data(textbook_data)" that's just called twice at the appropriate points?